scripts/qa_crowbarsetup.sh

```diff
@@ -1254,6 +1254,9 @@ function onadmin_allocate()
         done
         wait
     fi
+    if [[ $cloud = qa1 ]] ; then
+        curl http://clouddata.cloud.suse.de/git/automation/scripts/qa1_nodes_reboot | bash
```

Comment:
:-1: 
rather like this (thats exactly what we have these variables for, please use them):
```
${mkcloud_dir}/qa1_nodes_reboot
 or
bash -x ${mkcloud_dir}/qa1_nodes_reboot
```